### ISSUE TITLE
DTED2 Not Loading in SDK

### ISSUE THREAD

**Issue by tareed** [deptofdefense/AndroidTacticalAssaultKit-CIV#195]
I have an issue getting DTED2 to load properly in 4.3.0 and 4.4.0 SDK.  It works properly in 4.3.0 and 4.4.0 CIV.  In the SDK versions the tile is off by one degree to the east (see screenshots)

**4.4.0.15 - DEV**
![TAK_44015_DEV_DTED2](https://user-images.githubusercontent.com/1585286/157516405-6d8e4981-a441-4ff2-938a-31ad13bb1898.png)
.

**4.4.0 - CIV**
![TAK_4400_CIV_DTED2](https://user-images.githubusercontent.com/1585286/157516469-6e1cc544-8b15-4eb7-a20f-56bc2f76fcee.png)

**Comment by bisgroves** [comment]
When you say SDK, are you talking about the source code from https://github.com/deptofdefense/AndroidTacticalAssaultKit-CIV

**Comment by talentedbrute** [comment]
Yes.  Using the SDK and when using a custom build from the GitHub baseline.  Both have the same results.
